lib/internal/Magento/Framework/App/Feed.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Framework\App;
+
+class Feed implements \Magento\Framework\App\FeedInterface
+{
+    /**
+     * @var \Zend_Feed_Abstract
+     */
+    private $feed;
+
+    /**
+     * @param \Zend_Feed_Abstract $feed
+     */
+    public function __construct(\Zend_Feed_Abstract $feed)
```

Comment:
Please provide supported formats argument, configurable through DI.
As described below, to have a possibility verify whether supported format passed to getFormatedContentAs method